src/split-object/split-object-reinaldo/split-object-reinaldo.spec.js

```diff
@@ -0,0 +1,26 @@
+import { splitObjectReinaldo } from './split-object-reinaldo.js';
+
+describe('creating an array of objects, from an object', () => {
+  it('testing with numbers', () => {
+    const actual = splitObjectReinaldo({ a: 1, b: 2, c: 3 });
+    expect(actual).toEqual([{ a: 1 }, { b: 2 }, { c: 3 }]);
+  });
+  it('testing with different values', () => {
+    const actual = splitObjectReinaldo({
+      name: 'robs',
+      age: 25,
+      tall: true,
+      userName: 'sbor',
+    });
+    expect(actual).toEqual([
+      { name: 'robs' },
+      { age: 25 },
+      { tall: true },
+      { userName: 'sbor' },
+    ]);
+  });
+  it('testing with an empty object', () => {
+    const actual = splitObjectReinaldo({});
+    expect(actual).toEqual([]);
+  });
+});
```

Comment:
1. You have just 3 test cases and there should be at least 8 test cases.
2. The test cases must be grouped logically using describe. (That is something that could be improved too).
2.1. You need to explicit the side effects in the unit test. ;) Side-effects like: **returns a new array** and **the argument object is not modified**.

Reply:
Ok, done the 8 tests